src/parsing/Parse_rule.ml

```diff
@@ -851,7 +851,7 @@ let find_formula env (rule_dict : dict) : key * G.expr =
   | None ->
       error env rule_dict.first_tok
         "Expected one of `pattern`, `pattern-either`, `patterns`, \
-         `pattern-regex`, `pattern-comby` to be present"
+         `pattern-regex`, `pattern-coby` to be present"
```

Comment:
why this change?

Reply:
that was a mistake from when I was testing why the output wasn't what I expected, good catch